src/main/java/com/nina/modelling/shoes/ShoeFactory.java

```diff
@@ -0,0 +1,39 @@
+package com.nina.modelling.shoes;
+
+
+public class ShoeFactory extends Shoe {
+
+    public static void main(String[] args) {
+
+        KidShoe kidShoe = new KidShoe("Summer", "Flower", "Orthopedic");
+        KidShoe kidShoe1 = new KidShoe("Summer");
```

Comment:
Avoid using numbers in variable names.
Name it johnsShoe or marysShoe or shoeOnTheWindow

Reply:
working on it

Reply:
Changes the name of variables

Reply:
not all variables fixed

Reply:
All names of variables was changed.